docker-compose.yml

```diff
@@ -30,6 +30,9 @@ services:
       READ_MODEL_SPARQL_UPDATE_ENDPOINT: http://fuseki:3030/data-cube-read/update
       STORAGE_FILESYSTEM_PATH: /uploads
       NODE_ENV: production
+      AUTH_JWKS_URI: https://auth.lindas.admin.ch/realms/ref/protocol/openid-connect/certs
```

Comment:
I'd discover that based on the issuer metadata -> e.g. `fetch(AUTH_ISSUER + '/.well-known/openid-configuration').then(r => r.json()).then(m => m.jwks_uri)`

Reply:
hm, unfortunately the setup has to be done synchronously to create the vuex module.

Reply:
You only need that in the backend, not the frontend. The `secret` options in the JWT middleware is a callback so that should be doable